examples/all-clusters-app/cc13x2x7_26x2x7/BUILD.gn

```diff
@@ -26,17 +26,18 @@ import("${ti_simplelink_sdk_build_root}/ti_simplelink_sdk.gni")
 
 assert(current_os == "freertos")
 
+declare_args() {
+  device_vid = "0xFFF1"
```

Comment:
I could add that prefix, but these values are consumed by the application and not the matter configuration files. Then it's used by the TI executable targets. I noticed all other platforms just documented running the `ota_image_tool.py` again with different versions, but that didn't match the version in the persistent storage.

I'll check to see if it can be an int type. This eventually gets placed in a `#define` on the command line so that's my only concern.